ngraph/python/tests_cpp/mock_py_ngraph_frontend/mock_py_frontend.hpp

```diff
@@ -0,0 +1,542 @@
+// Copyright (C) 2018-2021 Intel Corporation
```

Comment:
I still disagree with the location of these test tools.
From my point of view (in my ideal world) we should have only two folders:

- src - with the code of component
- test - with all test cases and test tools

I don't fully understand why do you need to move test tools on the top level.

Reply:
It didn't look good for me, because currently 'tests' have only python files, thus I wanted to have mock frontends somewhere else. But ok, I've moved it to 'tests/mock' now, let's see if CI is good with that